[ZBX-20754] fixed Zabbix sender and Zabbix js not releasing semaphores
            upon exit (vso)

Index: src/zabbix_js/zabbix_js.c
--- src/zabbix_js/zabbix_js.c.orig
+++ src/zabbix_js/zabbix_js.c
@@ -249,6 +249,9 @@ int	main(int argc, char **argv)
 	printf("\n%s\n", result);
 close:
 	zabbix_close_log();
+#ifndef _WINDOWS
+     zbx_locks_destroy();
+#endif
 clean:
 	zbx_free(result);
 	zbx_free(error);
